src/components/ContentContainer/ContentContainer.tsx

```diff
@@ -1,12 +1,13 @@
 import { makeStyles } from '@material-ui/core';
 import React, { FC } from 'react';
 
-const useStyles = makeStyles((theme) => ({
+const useStyles = makeStyles(({ spacing }) => ({
   root: {
     flex: 1,
-    overflowY: 'scroll',
-    padding: theme.spacing(4),
-    maxWidth: theme.breakpoints.values.md,
+    overflow: 'hidden',
+    padding: spacing(4, 0, 0, 4),
+    display: 'flex',
+    flexDirection: 'column',
```

Comment:
This seems effective for the problem we're trying to solve, but I think we need to add some styling to `ProjectFilesList` to counteract this:

![image](https://user-images.githubusercontent.com/728777/93416689-440fc380-f874-11ea-9ef0-0eb1bd84ebb0.png)

Reply:
It's supposed to look like this:

![image](https://user-images.githubusercontent.com/728777/93416747-63a6ec00-f874-11ea-9a20-62a49a6445d1.png)

Reply:
Could just be as simple as wrapping the table in a `Grid`

Reply:
I just gave the table some padding on the right and bottom
![Screen Shot 2020-09-18 at 1 54 21 PM (2)](https://user-images.githubusercontent.com/43487134/93644343-a53fb000-f9b6-11ea-8f62-aa2d19c1e226.png).  Don't think we need Grid for this since it's just a single item.  Let me know what you think

Reply:
What I was thinking with the `Grid` was that we ought to keep it at that 2/3-ish width it was at before... thought I remembered that being something @CarsonF and I discussed, but maybe not. Care to weigh in, Carson?